content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -63,6 +63,9 @@ Ask your cluster administrator or consult your cloud provider or distribution do
 to determine if any sources of voluntary disruptions are enabled for your cluster.
 If none are enabled, you can skip creating Pod Disruption Budgets.
 
+**Not all voluntary disruptions are constrained by Pod Disruption Budgets. For example,
```

Comment:
```suggestion
{{< caution >}}
Not all voluntary disruptions are constrained by Pod Disruption Budgets. For example,
```

{{< warning >}}
--
15 | By the time you've reached this step you should have backed up your workloads and pertinent data; this section is for the complete destruction of a cluster.
16 | {{< /warning >}}


{{< warning >}}
--
15 | By the time you've reached this step you should have backed up your workloads and pertinent data; this section is for the complete destruction of a cluster.
16 | {{< /warning >}}